dotnet/src/Plugins/Plugins.Core/MathPlugin.cs

```diff
@@ -21,6 +21,7 @@ public sealed class MathPlugin
     /// <param name="amount">The amount to add as a string.</param>
     /// <returns>The resulting sum as a string.</returns>
     [SKFunction, Description("Adds an amount to a value")]
+    [return: Description("Summation of two values.")]
```

Comment:
nit: the description sounds more like the function description. I would expect the description of the return parameter to be something like "The resulting value".

Reply:
I believe "The resulting value" is too generic. If someone told me the return of a function was "the resulting value", I would say "of course" and then ask, "the resulting value of doing what?"  On the other hand, "summation" is the technical term for the result of adding two values.

Additionally, (yes a pun), this can be updated later to whatever works best for the planners.